types/react-native-sqlite-storage/index.d.ts

```diff
@@ -55,17 +61,28 @@ export interface SQLError {
 export type StatementCallback = (transaction: Transaction, resultSet: ResultSet) => void;
 export type StatementErrorCallback = (transaction: Transaction, error: SQLError) => void;
 export interface Transaction {
+    executeSql(sqlStatement: string, arguments?: any[]): Promise<TransactionStatementPromise>;
     executeSql(sqlStatement: string, arguments?: any[], callback?: StatementCallback, errorCallback?: StatementErrorCallback): void;
 }
 
 export type TransactionCallback = (transaction: Transaction) => void;
 export type TransactionErrorCallback = (error: SQLError) => void;
+export type StatementPromise = [ResultSet];
+export type TransactionStatementPromise = [Transaction, ResultSet];
 
```

Comment:
Same here.

Reply:
@dryganets same above, but in the TransactionStatementPromise the resulting array has 2 objects with different types.

`TransactionStatementPromise` should be `TransactionResultSetPromise`